### DISCUSSION TITLE
Scipy conference July 8-14 Tacoma, WA

### DISCUSSION THREAD

**Discussion by francisco-dlp** [hyperspy/hyperspy#3296] [General]
With the 2.0 Release, HyperSpy is now fully free from its microscopy bias. Therefore, I think that the time a ripe to submit an abstract to the Scipy Conference (deadline 27th of February). Is there anybody interested in presenting a talk on HyperSpy at the Scipy Conference?

**Comment by CSSFrancis** [comment]
@francisco-dlp I think this is a great idea. (Just because I couldn't find the website easily [here](https://www.scipy2024.scipy.org/) is the link.

I probably won't be able to attend (still trying to figure out a job/post graduation plans but I might be able to figure something out before Feb 27th)  

In general we probably need some better advertising and need to start to bring in some more people to work on the project.  Maybe we should talk a bit more about funding/ next steps etc.

**Comment by ericpre** [comment]
Moving to GitHub discussions.

**Comment by ericpre** [comment]
I don't think that I will go to SciPy 2024 and hopefully @CSSFrancis, you will be able to go! 😃

**Comment by CSSFrancis** [comment]
Hijacking this thread a bit but maybe we should just think about where people can/should present:

For example we have:

EMAS: May 12-15 
Microscopy And Microanalysis: July 28 – August 1
Euroscipy: (I think this is to be announced?)
Scipy: July 8-14th

It might be worth making 1 (or 2) presentations for HyperSpy 2.0.0 for both a microscopy audience and a more programming/technical audience and then having whoever can, present the presentation.

Something to note is that especially if we submit a Hyperspy 2.0.0 paper to M&M it will get cited even if we don't want it to :) The solution would be to maybe submit a Hyperspy 2.0.0 paper or maybe citing the M&M abstract will be fine.

**Reply by jlaehne** [reply]
For the records, I will present a HyperSpy/LumiSpy/eXSpy tutorial at the eBEAM summer school (Sept 1-13) in France together with Sean Collins.
I'll aim for other opportunities within the German community when they open up.

If it helps, my slides from last years CLEBIC workshop (15 min intro) can be found here: https://clebic2023.sciencesconf.org/resource/page/id/11
(Based on a LaTeX beamer template, happy to share the source and images if it helps - based on that I wanted to built a 45-60 min intro for seminars and the summer school)

**Reply by jlaehne** [reply]
Oh, and we do have an invited talk in the CL session (A1) at M&M, which my student Mikel Goméz Ruiz will be holding:
"Spatial, Spectral and Time Resolution: Tackling the Challenges of Multidimensional Luminescence Data Analysis with LumiSpy"

So there will be some HyperSpy represented in this context as well (but he'll also talk about a specific study and our new lab facilities).

**Comment by jat255** [comment]
I *may* be able to attend SciPy (I've always wanted to go, but have yet to). I'd have to do some checking, but I think it should be doable. If so, I'd be happy to present something put together by committee.

**Comment by jat255** [comment]
I received tentative approval to go to SciPy, so if this is something we want to put together, I think I should be able to present.

**Reply by CSSFrancis** [reply]
@jat255 That sounds great!  It looks like we would need a 100 word abstract and ~ 500 word description of the talk. There is specifically an image processing and visualization session which seems perfect!

**Reply by jat255** [reply]
@CSSFrancis Would you have some time to work on this with me on Monday? Submissions are due on Tuesday next week (sorry for sitting on this)

**Reply by CSSFrancis** [reply]
@jat255 sure! I don't have anything planned for Monday so I'm free all day. Do you want to send a meeting link?

**Reply by jat255** [reply]
Sounds good! I'll email you

**Comment by jat255** [comment]
@francisco-dlp @ericpre @CSSFrancis @jlaehne, I've emailed you a draft abstract (let me know if you have any issues accessing). The deadline was extended to March 5, so we have a little bit of time to hack on it before submitting.

I should point out that I had forgotten @to266 presented for us at SciPy 2016! (https://github.com/hyperspy/hyperspy/issues/928 and https://www.youtube.com/watch?v=kVlf3bMZcsc&list=PLYx7XA2nY5Gf37zYZMw6OqGFRPjB1jCy6&index=28), so this would not be our first time (although a lot has happened in 8 years :wink: )

**Comment by jat255** [comment]
Submitted! Thanks everyone

**Comment by jat255** [comment]
Following up on this, I received notification that our abstract was accepted and I received approval to travel to Tacoma for the conference, so HyperSpy will be presented! I might ask for some help putting the slides together as we get closer to July.

**Reply by CSSFrancis** [reply]
@jat255 That sounds great. Do we want to write up an associated paper?  I can work on some of that if you want.  It should translate to a nice couple of pages in my Thesis.

**Comment by jat255** [comment]
@hyperspy/developers closing the loop on this, the videos from the SciPy presentations are up, and the HyperSpy one is viewable here: https://www.youtube.com/watch?v=aObUhvtW6Ao&list=PL1PbeFStIOoO7rDLs431H-rn0h24Wr80S&index=18

**Reply by jlaehne** [reply]
How about you PR a News Item for the Website with a link to the video?

**Reply by jat255** [reply]
Good idea! I'll do that today

**Reply by jat255** [reply]
@jlaehne https://github.com/hyperspy/hyperspy-website/pull/68
